src/ScriptEditor/NetscriptDefinitions.d.ts

```diff
@@ -7991,79 +8065,6 @@ export interface NS {
    */
   vsprintf(format: string, args: any[]): string;
 
-  /**
-   * Format a number.
-   * @remarks
-   * RAM cost: 0 GB
-   *
-   * Converts a number into a numeric string with the specified format options.
-   * This is the same function that the game itself uses to display numbers. The format also depends on the Numeric
-   * Display settings (all options on the "Numeric Display" options page)
-   * To format ram or percentages, see {@link NS.formatRam | formatRam} and {@link NS.formatPercent | formatPercent}.
-   *
-   * This function has some quirky undocumented behaviors. This is a non-exhaustive list of those behaviors:
```

Comment:
You dropped this part. It makes me worried that you're missing other stuff that was updated in the docs.